Irina/src/main/java/oop/Item.java

```diff
@@ -0,0 +1,48 @@
+package oop;
+
+import java.util.Random;
+
+public class Item {
+    private long id;
+    private String title;
+    private Type type;
+    private int price;
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public Type getType() {
+        return type;
+    }
+
+    public void setType(Type type) {
+        this.type = type;
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    public void setPrice(int price) {
+        this.price = price;
```

Comment:
Please add some logic here to avoid setting negative price!

Reply:
Done